index.html

```diff
@@ -1,12 +1,148 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Just a suggestion, you can also add custom meta tags for eg, `<meta name="author" content="Your Name">` to your codebase a personal touch.